grunttasks/scan-email-translations.js

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('scan-email-translations', 'Scan the generated emails for invalid style tags', [
+    'generate-valid-styles',
+    'verify-styles'
+    ]
+  );
+
+  grunt.registerMultiTask('verify-styles', 'Verify whether the translated styles are valid', function (){
+    var fileArray = this.files[0].src;
+    var contents = grunt.file.read(fileArray.shift());
+    var styleArray = JSON.parse(contents);
+
+    var hasErrors = 0;
+    var styleRegex = /style='(.*)'/ig;
```

Comment:
> For example, I was curious why I could completely edit the style attribute in the French locale and it wouldn't detect it, 

Just to confirm, we must match on everything in the  style attribute, i.e the name "style" and all of its content.